### DISCUSSION TITLE
请问用哪套函数可以实现ppt上的时间表达？

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#31] [Q&A] [answered]
通过上网查询，我找到了如下的获取时间方式
time_t timep;
	time(&timep);
	char nowtime[30];
	strcpy(nowtime,ctime(&timep));
但是得到的时间格式不符合ppt上的要求，请问什么函数可以得到ppt上的时间表达呢

**Comment by binbin00** [comment, marked as answer]
command.txt只是查阅用的，不要求时间格式一模一样，只要求精确到秒。

**Comment by syhien** [comment]
或许你要找的是`std::put_time()`
